pkg/asset/manifests/dns.go

```diff
@@ -58,6 +71,23 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 		},
 	}
 
+	switch installConfig.Config.Platform.Name() {
+	case awstypes.Name:
+		zone, err := icaws.GetPublicZone(installConfig.Config.BaseDomain)
+		if err != nil {
+			return errors.Wrapf(err, "getting public zone for %s", installConfig.Config.BaseDomain)
+		}
+		config.Spec.PublicZone = &configv1.DNSZone{ID: strings.TrimPrefix(*zone.Id, "/hostedzone/")}
+		config.Spec.PrivateZone = &configv1.DNSZone{Tags: map[string]string{
+			"openshiftClusterID":                                                          clusterID.ClusterID,
+			fmt.Sprintf("kubernetes.io/cluster/%s", installConfig.Config.ObjectMeta.Name): "owned",
+			"Name": fmt.Sprintf("%s_int", installConfig.Config.ObjectMeta.Name),
```

Comment:
Does the format need to be `%s-int` to match the name set for the api_internal load balancer?
https://github.com/openshift/installer/blob/17c0fa8d093193fb933b476fd70f7c627520bd6f/data/data/aws/vpc/master-elb.tf#L1-L2

Reply:
No this is to match https://github.com/openshift/installer/blob/17c0fa8d093193fb933b476fd70f7c627520bd6f/data/data/aws/main.tf#L116

Reply:
Ah, yes. I couldn't find that last night for whatever reason.